src/main/java/com/it4045/pokemonviewer/enterprise/dto/Evolution.java

```diff
@@ -7,7 +7,7 @@
 class Evolution {
 
     @SerializedName("num")
-    private String evolutionNumber;
+    private int evolutionNumber;
     @SerializedName("name")
     private String pokemonEvolutionName;
```

Comment:
I itch at this one a bit as well.
Pokemon: in the package, this is a given.
Evolution: the name of the class; also a given.

Can this just be called name?